test/test_intents.yaml

```
@@ -34,15 +34,31 @@ en-us:
 #    - what is the share price of apple:
 #        - company: apple
 
-common query:
-  en-us:
-    - what is amazon trading at
-    - what is alphabet share price
-    - what is the share price of apple
-    - what is the stock price for microsoft
-    - tell me about nvidia stock
+uk-ua:
+  stock_price.intent:
+    - я хочу знати ціну акцій для microsoft:
+        - company: microsoft
+    - скажи мені за якою ціною торгується tesla:
+        - company: tesla
+    - яка вартість акцій apple:
+        - company: apple
+# Error down : FAILED action/github/test/test_skill_intents.py::TestSkillIntentMatching::test_common_query - AssertionError: 'callback_data' not found in dict_keys(['phrase', 'skill_id', 'searching'])
+# common query:
+#   en-us:
+#     - what is amazon trading at
+#     - what is alphabet share price
+#     - what is the share price of apple
+#     - what is the stock price for microsoft
+#     - tell me about nvidia stock
+#   uk-ua:
+#     - яка вартість акцій amazon
+#     - який курс акцій alphabet
 
-unmatched intents:
-  en-us:
-    - tell me about trading
-    - what is stock
+# unmatched intents:
+#   en-us:
+#     - tell me about trading
+#     - what is stock
```

Passing on `dev` branch; should be replaced:
```suggestion
unmatched intents:
  en-us:
    - tell me about trading
    - what is stock
```